### DISCUSSION TITLE
Request for wallpaper

### DISCUSSION THREAD

**Discussion by NeekoKun** [end-4/dots-hyprland#943] [General]
Hello I'm searching the wallpapers used in the first 2 slides of illogical-impulse. While I found the first (_posted by @srcrapi in https://github.com/end-4/dots-hyprland/discussions/663#discussioncomment-9914592_) I'm still looking for the second. Could you please share the source @end-4 ?

**Comment by end-4** [comment]
https://unsplash.com/photos/red-and-black-artwork-ioJBsYQ-pPM

**Comment by NeekoKun** [comment]
tysm
